src/ui/components/common/CopyButton.tsx

```diff
@@ -16,18 +16,27 @@ interface Props extends React.HTMLAttributes<unknown> {
 
 export function CopyButton({ className, iconClassName, value, id }: Props) {
   const ref = useRef<HTMLButtonElement>(null);
-  const onClick = useCallback((): void => {
-    copy(value);
+  const [showTooltip, setShowTooltip] = useState(false);
+  const onClick: MouseEventHandler = useCallback(
+    (event): void => {
+      event.stopPropagation();
 
```

Comment:
just a nit: can we remove the empty lines in this click handler? other than that LGTM

Reply:
✅